.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
Looks like this enables unstable tokio features also in release builds. Is that a good idea?

Reply:
"unstable features" doesn't mean the features are unstable, it means there are unstable APIs and trace events that are accessible with the flag enabled. Nothing in lemmy actually depends on the APIs, only `console-subscriber` consumes trace events, and those changing won't actually break the main application (although it will break console). IMO this is fine

Reply:
Alright then. But it has some conflicts, looks like you need to rebase again.

Reply:
Lol you're fast :D